File: Golang/summation/summation.go
package main

import "fmt"

func main() {
	var panjang int
	var sum int = 0

	fmt.Print("Masukkan jumlah angka yang ingin dijumlahkan: ")
	fmt.Scan(&panjang)
	
	temp := make([]int, panjang)

	for i := 0; i < panjang; i++ {
		fmt.Printf("Angka ke-%d: ", (i + 1))
		fmt.Scan(&temp[i])
		sum += temp[i]
	}
	fmt.Println(sum)
}

File: JavaScript/summation.js
const prompt = require("prompt-sync")({ signint: true });

let panjang;
let sum = 0;

panjang = parseInt(prompt("Masukkan jumlah angka yang ingin dijumlahkan: "));

let temp = [panjang];

for (let i = 0; i < panjang; i++) {
	temp[i] = parseInt(prompt("Angka ke-" + (i + 1) + ": "));
	sum += temp[i];
}
console.log(sum);
